src/functions-and-arrays.js

```diff
@@ -52,16 +82,25 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(wordsUniqueArray) {
 
+}
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
-
-
+function doesWordExist(wordsFindArray,word) {
+  if (wordsFindArray.length === 0) {
+    return null;
+  }
+  let found = false
+  for (let i = 0; i < wordsFindArray.length; i++){
+    if (wordsFindArray [i] === word){
+      found = true
+    }
+  }
+  return found
+}
```

Comment:
This works but if the target word is the first word in the array and the array has 1000 elements, it will still loop on the entire array before returning true. Instead you could directly return true in line 99 so the function ends and return true as soon as the target is found. If the for loop is finished, then this means the target was not found and you can return false.